src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -96,9 +96,9 @@ public sealed class Features
         /// </summary>
         internal bool IsUserDefinedTypesEnabled { get; set; } = false;
 
-        internal static Features None => new Features();
+        internal static readonly Features None = new Features();
 
-        public static Features PowerFxV1 => new Features
+        public static readonly Features PowerFxV1 = new Features
```

Comment:
The instances are readonly, but now it's possible that someone inadvertently change the meaning of those properties (i.e., by doing something like `Features.None.IsUserDefinedTypesEnabled = true`)

Reply:
If we are to do this change, we should change this class to be immutable, and add a constructor where we can toggle those values.

Reply:
Yup, changing set to init on all these